### ISSUE TITLE
iOS: UNKNOWN_ERROR when adapty.paywalls.getPaywalls() 

### ISSUE THREAD

**Issue by pvshum** [adaptyteam/AdaptySDK-React-Native#7]
Hi There,
I am facing the UNKNOWN_ERROR when fetching paywalls.
Any insights how to debug or get more meaningful output?

**platform**
iOS

**code**
```
    const activate = await activateAdapty({
      sdkKey: 'public_live_ my key ', 
      logLevel: "verbose"
    });
    try {
      await adapty.paywalls.getPaywalls({ forceUpdate: true })
    } catch (error) {
      console.error(error)
    }
```

**output**
ERROR
```
{"adaptyCode": "unknown", "code": 0, "localizedDescription": "UNKNOWN_ERROR"}
```

**react-native info**
```
System:
    OS: macOS 11.4
    CPU: (8) x64 Intel(R) Core(TM) i5-1038NG7 CPU @ 2.00GHz
    Memory: 397.40 MB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 14.15.5 - ~/.nvm/versions/node/v14.15.5/bin/node
    Yarn: 1.22.10 - ~/.nvm/versions/node/v14.15.5/bin/yarn
    npm: 6.14.11 - ~/.nvm/versions/node/v14.15.5/bin/npm
    Watchman: 2021.08.30.00 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.10.1 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: iOS 14.5, DriverKit 20.4, macOS 11.3, tvOS 14.5, watchOS 7.4
    Android SDK:
      API Levels: 29, 30
      Build Tools: 28.0.3, 29.0.2, 30.0.3
      System Images: android-29 | Intel x86 Atom, android-30 | Google APIs Intel x86 Atom
      Android NDK: Not Found
  IDEs:
    Android Studio: 4.1 AI-201.8743.12.41.7042882
    Xcode: 12.5.1/12E507 - /usr/bin/xcodebuild
  Languages:
    Java: 1.8.0_282 - /usr/bin/javac
    Python: 2.7.16 - /usr/bin/python
  npmPackages:
    @react-native-community/cli: Not Found
    react: 16.13.1 => 16.13.1
    react-native: 0.63.3 => 0.63.3
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```

```
"react-native-adapty": "^1.0.3",
```

**Comment by Betra** [comment]
Hey! Do you use iOS simulator? 
Since iOS 14+ Apple restricted a usage of StoreKit SDK for simulators. 
If this your case you can either make your simulator iOS13 or use a real device. 

Sorry for an inconvenient error message, we are trying hard to implement most uncostly solution to track this problem

**Comment by pvshum** [comment]
@Betra I switched to test on the real device and the error is gone, thank you!
There's a different problem now `{"adaptyCode": "noProductsFound", "code": 1001, "localizedDescription": "No In-App Purchases were found."}` however products, paywalls etc are set up in Adapty dashboard and in AppStore (Ready to Submit)
Any insights on how to debug this?

Original issue is resolved and can be closed though!

**Comment by pvshum** [comment]
In case anyone interested about `noProductsFound` error when having everything set up in both AppStore Connect and Adapty dashboard. Here's useful info: [link](https://community.revenuecat.com/sdks-51/why-are-offerings-or-products-empty-124) 
TL;DR: Make sure you've signed your 'Paid Applications Agreement' in AppStore Connect.
